skribenten-web/frontend/src/Brevredigering/LetterEditor/actions/table.ts

```diff
@@ -80,6 +84,11 @@ export const removeTable = produce<LetterEditorState>((draft) => {
 
   const parentBlock = draft.redigertBrev.blocks[blockIndex];
   removeElements(contentIndex, 1, parentBlock);
+
```

Comment:
Kan vi ende opp med en tom blokk her? Hva er konsekvensen av det, f.eks vil det være mulig for bruker å skrive noe i den blokken?